packages/did-ethr-resolver/src/implementations/resolver.ts

```diff
@@ -13,11 +13,18 @@ import {
   KeyTags,
   DocumentSelector,
 } from '@ew-did-registry/did-resolver-interface';
-import { Methods, DIDPattern } from '@ew-did-registry/did';
+import {
+  Methods,
+  DIDPattern,
+  ethAddrPattern
+} from '@ew-did-registry/did';
 import { ethrReg } from '../constants';
 import { fetchDataFromEvents, wrapDidDocument, query } from '../functions';
 import { compressedSecp256k1KeyLength } from '..';
 
+// temporary declaration, will use as imported dependency from new release
```

Comment:
Is there is something prevents to move did-related code to did package now?

Reply:
I was getting error after build if I tried importing it.